#goal definition
string skill_command
uint16 max_time
uint16 number_plays
uint8 proactivity
string skill_status
---
#result definition
int8 ERROR = -1
int8 SUCCESS = 0
int8 FAIL = 1
int8 skill_result
---
#feedback definition
string app_status
uint8 percentage_completed
bool engagement
